libgalois/include/katana/MemorySupervisor.h

```diff
@@ -38,32 +45,32 @@ class KATANA_EXPORT MemorySupervisor {
     return mm_;
   }
 
-  /// Inform MS of allocation of \p bytes for active memory
-  /// Application cannot continue if it does not get memory
-  void BorrowActive(const std::string& name, count_t bytes);
   /// Request permission to allocate \p bytes for standby memory
   /// Returns the number of bytes granted, possibly 0
   count_t BorrowStandby(const std::string& name, count_t goal);
 
-  /// Notify MS that manager freed \p bytes of active memory.
-  void ReturnActive(const std::string& name, count_t bytes);
   /// Notify MS that manager freed \p bytes of standby memory.
   void ReturnStandby(const std::string& name, count_t bytes);
```

Comment:
For some reason I expected this function to return some value. Perhaps the name is a little misleading.

Reply:
That is a good point.  I was going for `Borrow/Return`.  I changed it to `Get/Put`.